env-var-docs/parser-package/README.md

```diff
@@ -92,7 +92,8 @@ Here is an example environment variable documentation file (validated in
         "members": {
             "TITLE": {
                 "description": "Sets the CiviForm title.",
-                "type": "string"
+                "type": "string",
```

Comment:
suggestion (blocking): please add `mode` + an explanation of the field to the "Variables have the following fields:" section above. Since mode is a required field, please also add it to this sentence: "The `description` and `type` fields must be defined for each variable." 

I think you should also add an "is not None" check for Mode in `parser.py` (beneath the "# appease mypy" comment), although I'm not positive because I'm not totally clear on why we need that.

Reply:
Done